### ISSUE TITLE
[Bug]: Cannot Stop or Reverse Cover Operation Mid-Action via Alexa

### ISSUE THREAD

**Issue by ohrenmasken** [t0bst4r/home-assistant-matter-hub#328]
### Version

3.0.0-alpha.55

### Matter Controller

Alexa

### Steps to reproduce

I successfully installed the add-on and can control my covers through Alexa. For example, saying, “Alexa, open the living room cover” opens the cover as expected. However, if I say, “Alexa, close the living room cover” while the cover is still opening, the command is ignored.

I can only use the “close” command after the cover has fully opened. This makes it inconvenient if I accidentally open a cover and want to close it immediately.

Steps to Reproduce:
Install and set up the add-on.
Use Alexa to open the cover by saying, “Alexa, open the living room cover.”
During the opening process, say, “Alexa, close the living room cover.”

Expected Behavior:
The cover should stop and begin closing when the “close” command is given, even if it is still in the process of opening.

Actual Behavior:
The “close” command is ignored until the cover is fully opened.

### Documentation & Issues

- [X] I have reviewed the [documentation](https://github.com/t0bst4r/home-assistant-matter-hub/blob/main/packages/documentation/README.md) and the linked troubleshooting guide.
- [X] I have searched the [issues](https://github.com/t0bst4r/home-assistant-matter-hub/issues) for a similar problem.

**Comment by t0bst4r** [comment]
This is only possible if Alexa is actually sending the close command while opening the cover. If that's an Alexa issue we are not able to do anything about it.

Can you please enable debug logging and reproduce the issue. Please attach the logs of this. If possible, only attach the logs of that two commands and not the last 5 days of logs 😅

**Comment by ohrenmasken** [comment]
I have now replicated the situation. The living room shutter was closed, and I opened it via Alexa. While it was opening, I tried three times to close the living room shutter. The commands can also be seen in the log, but the shutter continued to open.

```
[ 2024-12-20T06:52:46.911Z ] [ INFO  ] [ Matter / Behavior                                  ]: Invoke *86c79290a1eb4d0c967eda36b36d903a.aggregator.cover_wohnzimmer_rollladen.windowCovering.goToLiftPercentage* online#117f395@1a0f68b9003aeb1 liftPercent100thsValue: 0
[ 2024-12-20T06:52:57.125Z ] [ INFO  ] [ Matter / Behavior                                  ]: Invoke *86c79290a1eb4d0c967eda36b36d903a.aggregator.cover_wohnzimmer_rollladen.windowCovering.goToLiftPercentage* online#117f39b@1a0f68b9003aeb1 liftPercent100thsValue: 10000
[ 2024-12-20T06:53:03.612Z ] [ INFO  ] [ Matter / Behavior                                  ]: Invoke *86c79290a1eb4d0c967eda36b36d903a.aggregator.cover_wohnzimmer_rollladen.windowCovering.goToLiftPercentage* online#117f3a0@1a0f68b9003aeb1 liftPercent100thsValue: 10000
[ 2024-12-20T06:53:07.482Z ] [ INFO  ] [ Matter / Behavior                                  ]: Invoke *86c79290a1eb4d0c967eda36b36d903a.aggregator.cover_wohnzimmer_rollladen.windowCovering.goToLiftPercentage* online#117f3a5@1a0f68b9003aeb1 liftPercent100thsValue: 10000
```

**Comment by t0bst4r** [comment]
I was just testing that with my own (virtual) cover.
For me it works.
1. "Alexa, Rollladen öffnen"
2. waiting some seconds until its half-opened
3. "Alexa, Rollladen schließen"
4. it immediately starts closing.

Could it be a problem of your actual cover, that its not controllable while executing a command?
Can you change the target position mid-action using the Home Assistant UI?

**Comment by ohrenmasken** [comment]
I used the Shelly plugin for my roller shutter for over two years, and it worked perfectly. For the past six months, I’ve been using it with an Amazon Alexa Smart Home Skill, see https://www.home-assistant.io/integrations/alexa.smart_home/. It also worked flawlessly with that setup.

I just tried it now in Home Assistant:

	1.	The living room roller shutter is closed
	2.	I open the living room roller shutter via Home Assistant
	3.	I wait a few seconds until it is halfway open
	4.	I try to close the living room roller shutter via Home Assistant
	5.	It doesn’t work; the button is grayed out in toggle mode

The only thing that works in Home Assistant is if I open the shutter and then select 0% in position mode, it will close during the opening process.

So you're right. It seems to be a HA problem or a problem with my shutter. I have to investigate this.
